### ISSUE TITLE
ooniprobe debian package (1.5.1-1) wrong PATH in usage prompt

### ISSUE THREAD

**Issue by anadahz** [TheTorProject/ooni-probe#539]
Debian stretch, ooniprobe 1.5.1-1
------------------------------------------
After installing ooniprobe the uses is being instructed to run the default deck from a non existent PATH: 
```
Run ooniprobe like so:
ooniprobe -i /var/lib/ooni/decks/deck-XX/default-user.deck
Moving deck to /var/lib/ooni/decks/debian-default/user.deck
```
Perhaps we should add a line at the end pointing to the correct PATH?

Install log:
`# apt-get install ooniprobe`
```
[...]
Could not generate country specific url list
We will just use the global one.
Deck written to /var/lib/ooni/decks/deck-XX/default-user.deck
Run ooniprobe like so:
ooniprobe -i /var/lib/ooni/decks/deck-XX/default-user.deck
Moving deck to /var/lib/ooni/decks/debian-default/user.deck
It will be run daily through cron.
Setting up python-pypcap (1.1.2+debian-2.2) ...
Processing triggers for libc-bin (2.22-11) ...
```
`# ooniprobe -i /var/lib/ooni/decks/deck-XX/default-user.deck`

```
WARNING: running ooniprobe involves some risk that varies greatly
         from country to country. You should be aware of this when
         running the tool. Read more about this in the manpage or README.
Ooni home directory does not exist.
Creating it in '/root/.ooni'.
Configuration file does not exist.
Creating it for you in '/root/.ooni/ooniprobe.conf'.
Log opened.
[!] IOError: [Errno 2] No such file or directory: '/var/lib/ooni/decks/deck-XX/default-user.deck'
```
cc: @irl

**Comment by hellais** [comment]
I think the issue here is that the output of oonideckgen is not being suppressed and it leads to a slightly confusing output.

The line below though clearly says "Moving deck to /var/lib/ooni/decks/debian-default/user.deck", is the default deck not available at that path?

I think the best solution for this would be to redirect standard output of oonideckgen to `/dev/null`.

**Comment by anadahz** [comment]
@hellais yes this is the right PATH: `/var/lib/ooni/decks/debian-default/user.deck`.
This is very confusing to the user I mistakenly used the way instructed by oonideckgen and I guess this what most people will do.
Do you think that this could be better resolved by #502 with an option to display less output log (like oonideckgen usage).

**Comment by hellais** [comment]
@anadahz I think that since this is something specific to the behaviour that is done by the debian package post-install script (i.e. it moves the deck after it has created it), the problem should be addressed in the debian package rather than in ooniprobe itself.

I think the best solution to this is to redirect standard output to /dev/null.

**Comment by irl** [comment]
Ok, can do a revision of the package to update the post-install script.

Debian bug is now: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=827359

**Comment by irl** [comment]
Fixed in 1.5.1-2 by redirecting output to /dev/null.

Package uploaded to unstable.
